src/main/java/by/itech/lab/supplier/service/StateService.java

```diff
@@ -4,6 +4,11 @@
 import org.springframework.data.domain.Page;
 import org.springframework.data.domain.Pageable;
 
-public interface StateService {
+import java.util.List;
+
+public interface StateService{
```

Comment:
Is that application state? Clarify what the state is in the name.

Reply:
we think about it